src/System.Management.Automation/utils/PInvokeDllNames.cs

```diff
@@ -135,6 +135,7 @@ internal static class PinvokeDllNames
         internal const string CreateToolhelp32SnapshotDllName = "api-ms-win-core-toolhelp-l1-1-0";           /*120*/
         internal const string Process32FirstDllName = "api-ms-win-core-toolhelp-l1-1-0";                     /*121*/
         internal const string Process32NextDllName = "api-ms-win-core-toolhelp-l1-1-0";                      /*122*/
+        internal const string GetACPDllName = "api-ms-win-core-localization-l1-2-0.dll";                     /*123*/
```

Comment:
`GetACP` is in `api-ms-win-core-localization-l1-2-0.dll` of `runtime.win7-x64.Microsoft.NETCore.Windows.ApiSets` dump.
But It seems Nano use `runtime.win10-x64.Microsoft.NETCore.Windows.ApiSets`